ui/raidboss/data/05-shb/raid/e9n.txt

```diff
@@ -4,7 +4,64 @@
 hideall "--Reset--"
 hideall "--sync--"
 
+# -ii 5626 5B18 55E1 55E3 5159 532F 55DB 55E5
+
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
 
-0 "Start"
-0.0 "--sync--" sync /:Engage!/ window 0,1
+0 "Start" sync /Engage!/ window 0,1
+# Unfortunately we can't sync to a first attack here,
+# as Cloud of Darkness uses an auto at random before starting her 3-second rotation.
```

Comment:
Could sync to the starts casting line of Ground-Razing Particle Beam to start the timeline sooner, maybe?